share/lutris/ui/lutris-window.ui

```diff
@@ -614,6 +614,10 @@
                             <property name="can_focus">True</property>
                             <property name="halign">center</property>
                             <property name="valign">center</property>
+                            <property name="margin_left">5</property>
```

Comment:
I believe you only need one property: `<property name="margin">5</property>`

Reply:
Glade does not provide a way to change all the margin.
However when I edit the file in a text editor with your proposed change, Glade produces this preview, which misses a margin-top, and also can't display the change it it's "margin" config panel on the right side:

![image](https://user-images.githubusercontent.com/15329494/54369317-04ac5980-4676-11e9-9607-f96dd116cba0.png)

Reply:
Well glade isn't very useful most of the time and you can just edit the file directly.

But its not a big deal eitherway.

Reply:
I disagree ^.^ it's very helpful for designing :)

Reply:
I mean, in theory, but it doesn't actually understand the format its reading/writing often.

Reply:
It is a big deal to me, if Glade were usable, I would actually write .ui files instead of going for Python code, which is current preferred method. Really not a fan of editing XML files manually.